Makefile

```diff
@@ -548,7 +548,7 @@ sso/setup:
 .PHONY: sso/setup
 
 sso/config:
-	KEYCLOAK_URL=$(KEYCLOAK_URL) KEYCLOAK_PORT_NO=$(KEYCLOAK_PORT_NO) KEYCLOAK_USER=$(KEYCLOAK_USER) \
+	KEYCLOAK_URL=$(KEYCLOAK_URL) KEYCLOAK_USER=$(KEYCLOAK_USER) \
```

Comment:
This line:
https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/41ca4d657372a66cb53bcb5b80ef91a12fcb5300/Makefile#L541

will have to be `KEYCLOAK_URL ?= http://localhost:8180`